apps/x509.c

```diff
@@ -1037,7 +1037,8 @@ int x509_main(int argc, char **argv)
         goto end;
     }
 
-    print_cert_checks(out, x, checkhost, checkemail, checkip);
+    if (!check_cert_attributes(out, x, checkhost, checkemail, checkip, 1))
```

Comment:
Why do you explicitly change the behavior here? It could be optional, but not by default.

Reply:
See https://github.com/openssl/openssl/pull/17022

Reply:
OK, thanks!